README.md

```diff
@@ -329,6 +329,28 @@ class Product {
 }
 ```
 
+If you have a list of acceptable status codes you can also pass an array to the
+```statusCode``` option:
+
+```php
+<?php
+namespace CircleBundle\Entity;
+
+use Doctrine\ORM\Mapping as ORM;
+use Circle\DoctrineRestDriver\Annotations as DataSource;
+
+/**
+ * This annotation marks the class as managed entity:
+ *
+ * @ORM\Entity
+ * @ORM\Table("products")
+ * @DataSource\Select("http://www.yourSite.com/api/products/findOne/{id}", statusCode={200, 203, 404})
```

Comment:
It will still throw an error because the data type has changed from string to array